### DISCUSSION TITLE
Need a way to jump to other boards / back to project

### DISCUSSION THREAD

**Discussion by sschuberth** [community/community#6226] [Projects and Issues]
In the new issues / projects UI, there seems to be no way to quickly jump from a project view to another project view, or back to the code of the belonging Git project. Having such a quick navigation would be appreciated.

**Comment by topialla** [comment]
Definitely needed, so we can jump around. Maybe also directly from highlevel into the Project, the issue belongs to?

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
